### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by minotogna**
WIP #415 

- [X] Refactor login according to mockup
- [ ] ForgotPassword submit server error 500
- [ ] Refactor resetPassword

### CODE REVIEW

File: webapp/Login/LoginForm/ForgotPassword/ForgotPassword.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import Error from '../Error'
+
+import { resetPassword, resetPasswordFormReset } from '../../actions'
+
+const ForgotPassword = (props) => {
+  const { onClose } = props
+
+  const dispatch = useDispatch()
+  const emailRef = useRef(null)
+  const { error, message } = useSelector(R.pathOr({}, ['login', 'localLogin', 'resetPassword']))
```
Comment (sorja) [inline webapp/Login/LoginForm/ForgotPassword/ForgotPassword.js:15]: possible to use LoginState.getResetPassword or similar?

File: webapp/Login/LoginForm/LocalLogin/LocalLogin.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+import Error from '../Error'
+import ForgotPassword from '../ForgotPassword'
+
+const LocalLogin = (props) => {
+  const { invitation, user, onCancel } = props
+  const { invitationUuid } = invitation
+
+  const dispatch = useDispatch()
+  const message = useSelector(R.path(['login', 'localLogin', 'message']))
```
Comment (sorja) [inline webapp/Login/LoginForm/LocalLogin/LocalLogin.js:16]: Same as above

File: webapp/Login/LoginForm/LoginForm.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+import { getUrlParameter } from '@webapp/utils/urlUtils'
+
+import { initLogin } from '../actions'
+
+import Error from './Error'
+import LocalLogin from './LocalLogin'
+
+const LoginForm = () => {
+  const { status, invitation, user } = useSelector(R.pathOr({}, ['login', 'login']))
```
Comment (sorja) [inline webapp/Login/LoginForm/LoginForm.js:13]: Same as above

File: webapp/Login/components/resetPasswordForm.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  ) : R.isNil(status) ? null : (
+    <ResetPasswordNotFound />
+  )
+}
+
+const mapStateToProps = (state) => ({
+  status: R.path(['login', 'resetPassword', 'status'], state),
+  resetPassword: R.path(['login', 'resetPassword', 'data'], state),
+  changePasswordResponse: R.path(['login', 'changePassword'], state),
+})
```
Comment (sorja) [inline webapp/Login/components/resetPasswordForm.js:86]: UseSelector

File: webapp/Login/LoginForm/ForgotPassword/ForgotPassword.js (hunk 1 above)
Comment (minotogna) [reply webapp/Login/LoginForm/ForgotPassword/ForgotPassword.js:15]: there's another issue on the board. I mentioned this morning I wouldn't refactor the redux store in this issue

File: webapp/Login/components/resetPasswordForm.js (hunk 4 above)
Comment (minotogna) [reply webapp/Login/components/resetPasswordForm.js:86]: resetPasswordForm is the last item of the issue. not in this pull request. please check

**Pull request merged**